api/helpers/gamestate/moves/resolve/execute.js

```diff
@@ -0,0 +1,66 @@
+const GamePhase = require('../../../../../utils/GamePhase.json');
+const MoveType = require('../../../../../utils/MoveType.json');
+
+module.exports = {
+  friendlyName: 'Resolve One-Off',
+
+  description: 'Returns new GameState resulting from requested resolve move',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      description: 'The latest GameState before the requesting player resolves the pending one-off and counters',
+      required: true,
+    },
+    /**
+     * @param { Object } requestedMove - The move being requested. 
+     * Specifies that the move is to resolve
+     * @param { MoveType.RESOLVE } requestedMove.moveType
+     */
+    requestedMove: {
+      type: 'ref',
+      description: 'The move being requested. Specifies which player is asking to play a one-off ',
+    },
+    playedBy: {
+      type: 'number',
+      description: 'Player number of player requesting move.',
+    },
+  },
+  sync: true, // synchronous helper
+  fn: ({ currentState, requestedMove, playedBy }, exits) => {
+    let result = _.cloneDeep(currentState);
+
+    const { oneOff } = result;
```

Comment:
I believe that 2's and 9's will also need to grab the `targetCard` and `targetType` from the `result` before those values are set to null below, but that isn't used yet so eslint was erroring. We'll re-add them once 2 or 9 is put into place